pynicotine/plugins/core_commands/__init__.py

```diff
@@ -26,39 +26,637 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.commands = {
+            "help": {
+                "aliases": ["?"],
+                "callback": self.help_command,
+                "description": "List commands",
+                "usage": ["[query]"]
+            },
+            "rescan": {
+                "callback": self.rescan_command,
+                "description": _("Rescan shares"),
+                "group": _("Shares"),
+                "usage": ["[-force]", ""]
+            },
+            "hello": {
+                "aliases": ["echo", "greet"],
+                "callback": self.hello_command,
+                "description": "Print something",
+                "group": _("Message"),
+                "usage": ["[something..]"]
+            },
+            "away": {
+                "aliases": ["a"],
+                "callback": self.away_command,
+                "description": _("Toggle away status")
+            },
+            "plugin": {
```

Comment:
I think I'd prefer separate commands here.
- toggle-plugin
- enable-plugin
- disable-plugin

Reply:
The enable|disable functions act more like load and unload, because the setting is not permanently saved.

Reply:
I suppose we can use subcommands if they are simple enough.